watchtower/wtdb/client_chan_summary.go

```diff
@@ -3,9 +3,24 @@ package wtdb
 import (
 	"io"
 
+	"github.com/lightningnetwork/lnd/fn"
 	"github.com/lightningnetwork/lnd/lnwire"
 )
 
+// ChannelInfos is a map for a given channel id to it's ChannelInfo.
+type ChannelInfos map[lnwire.ChannelID]*ChannelInfo
+
+// ChannelInfo contains various useful things about a registered channel.
+type ChannelInfo struct {
+	ClientChanSummary
```

Comment:
nit: I wonder if it's worth adding a comment why the `ClientChanSummary` type is decorated vs extended. So future readers (if that's not you) don't get confused.

Reply:
good point - Ok i've updated it with a comment but im also not that opposed to extending ClientChanSummary since we do follow that sort of pattern for other things like `ChanEdgeInfo` etc. 

It just means that we have to decode and re-encode the struct a bunch.

What do you think?